app/services/notify_app_store.rb

```diff
@@ -3,10 +3,12 @@ class NotifyAppStore < ApplicationJob
 
   def self.process(claim:)
     if ENV.key?('REDIS_HOST')
-      set(wait: Rails.application.config.x.application.app_store_wait.seconds).perform_later(claim)
+      set(wait: Rails.application.config.x.application.app_store_wait.seconds)
+        .perform_later(claim)
     else
       begin
         new.notify(MessageBuilder.new(claim:))
+        ClaimFeedbackMailer.notify(claim).deliver_later!
```

Comment:
Is there a timing issue here? The feedback email will be delivered almost instantaneously, but the provider app only updates itself from the app store periodically, so will this lead to providers being told about an update to their claim, and then going to look at the provider app to see the specifics and there being nothing there?

Reply:
As it stands your right there is a timing issue. However there is a possibility that is being discussed/understood around delaying sending the email for a period of time to allow caseworkers to be able to change/undo their actions and recall the email. This isnt implemented now however so that we can check it is sent and that content is correct